libs/community/pyproject.toml

```diff
@@ -119,7 +120,7 @@ pytest-vcr = "^1.0.2"
 vcrpy = "^6"
 
 [tool.poetry.group.lint.dependencies]
-ruff = "^0.5"
+ruff = "~0.9.1"
```

Comment:
```suggestion
ruff = "^0.9.1"
```

Reply:
Ruff’s versioning policy is to add new rules and change formatting between minor versions. That’s why I think pinning the minor is better here.